soco/core.py

```diff
@@ -1287,6 +1288,35 @@ def add_to_queue(self, queueable_item):
         qnumber = response['FirstTrackNumberEnqueued']
         return int(qnumber)
 
+    def add_multiple_to_queue(self, item_list, container=None):
+        """Adds a list of items to the queue.
+
+        :param item_list: The list of items to be added to the queue
```

Comment:
We are in the process of migrating all doc strings to fit with the Napoleon extension for  Sphinx and its syntax (although it isn't always that obvious, because we are not that long into the process. But all new doc strings should follow it e.g:
```python
Args:
    items (list): A sequence of items to the be added to the queue
```
etc.